data/hai/hai missions.txt

```diff
@@ -1989,58 +2028,47 @@ mission "Strider 2 (Remnant)"
 			`	"We too have encountered these Korath," Yashili says in response. "Although, we had not taken their technology. This may be of great use to us."`
-			`	The Remnant agree to meet in the spaceport with a copy of the research done on the Korath technology, as well as a collection of the small robots for them to study.`
+			`	"We would like to ask something in trade, however. We would like a copy of your astronomical records. The history of the galaxy itself is a mystery to us, and we would like to learn more about it."`
+			`	Ambassador Yashili looks thoughtful, then nods. "Historical astronomical data is scientifically valuable, as is learning how to repair our ships in flight. This seems a fair trade." She nods to one of her aides who begins tapping out commands on a small computer. "We should be able to have the data for you shortly. My assistant will need to use Captain <first>'s ftl communication system, but it shouldn't be long."`
+			`	As the assistant scurries out, Ambassador Yashili and the Remnant settle into diplomatic small talk. The conversational dance between a group of professional diplomats and a secretive group that are currently playing the role of nomadic desert traders is an odd one.`
 
-mission "Strider 3 (Remnant)"
+
+
+mission "Strider: Remnant 3"
 	name "Technology Obtained"
 	description "The delegation has successfully obtained hull repair technology from the Remnant. Return to <destination> so that the Hai can begin developing this technology for their new ship."
-	source "Aventine"
+	landing
+	source "Deadman's Cove"
 	destination "Hai-home"
 	passengers 3
 	cargo "repair technology" 12
 	blocked "You need <capacity> in order to return the Hai delegation and the repair technology back to <planet>."
 	to offer
-		has "Strider 3 (Remnant): done"
+		has "Strider: Remnant 2: done"
 	on offer
 		conversation
-			`The Remnant hand the Hai a data chip containing a copy of all the information that they have on the Korath technology and load a crate full of samples of the Korath outfits onto your ship. Yashili thanks the Remnant for their generosity and boards your ship, ready to return to <destination>.`
+			`An hour later the ambassador's assistant returns with a large data drive, and hands it to Ambassador Yashili. The Remnant pull out a datachip of their own, and the two groups formally exchange the data. In response to an unseen signal, another Remnant pulls back one side of the tent and a team carries in a segment from a systems core with several of the small robots. "There is more of these waiting outside for you, but here is a sample for you to inspect. Using the instructions we have provided, you should have no problem making use of this technology." They poke at the console, and the robots promptly spring to life and assembled a small pile of parts into a model ship.`
```

Comment:
```suggestion
			`An hour later, the ambassador's assistant returns with a large data drive and hands it to Ambassador Yashili. The Remnant pull out a data chip of their own, and the two groups formally exchange the data. In response to an unseen signal, another Remnant pulls back one side of the tent, and a team carries in a segment from a systems core with several of the small robots. "There is more of these waiting outside for you, but here is a sample for you to inspect. Using the instructions we have provided, you should have no problem making use of this technology." They poke at the console, and the robots promptly spring to life and assemble a small pile of parts into a model ship.`
```
A couple spelling/grammar corrections. One thing of note: in-game files says "data cards", so I don't know if the space applies to "data chips" as well. I've added it to my suggestion so the question is at least asked.
Edit: Later on, you use "data chip", so I'll assume this correction is correct.